sos/plugins/kernel.py

```diff
@@ -81,7 +82,10 @@ def setup(self):
             "/proc/timer*",
             "/proc/lock*",
             "/proc/misc",
-            "/var/log/dmesg"
+            "/var/log/dmesg",
+            numa_path + "node*/meminfo",
```

Comment:
You can always just force-push an update to this branch - saves churning new PR#s.